cypress/README.md

```diff
@@ -0,0 +1,114 @@
+## Install
+
+You need to have the project up and running on port 4200 ⇒ https://github.com/usealto/assessment-front/blob/develop/README.md
```

Comment:
the `main` branch would be better here

Reply:
also, you could make something like [see README](https://github.com/usealto/assessment-front/blob/develop/README.md)

Reply:
done